src/test/java/uk/ac/ebi/ena/webin/cli/WebinCliBuilder.java

```diff
@@ -68,6 +68,24 @@ private WebinCliCommand cmd(Path inputDir, ManifestBuilder manifestBuilder) {
     return cmd;
   }
 
+  private WebinCliParameters params(Path inputDir, ManifestBuilder manifestBuilder) {
```

Comment:
- ```WebinCliParameters``` should only be created in ```WebinCli.initParameters``` (except in unit tests).
- ```params``` functionality is called only from tests.
-  We should not add test only functionality into WebinCliBuilder.
- Please remove ```params``` from WebinCliBuilder.
- Why not just call ```WebinCli(WebinCliParameters parameters)``` from unit tests instead of using ```WebinCliBuilder```.